### PR TITLE
Update README with environment suggestions

### PR DESCRIPTION
**Pull request by martinacantaro**
To help onboard new teammembers, especially now that there's rotations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ Curse of Myrra is a game that uses the [Lambda backend game engine](https://gith
 - Unity
 - Docker
 
+## Suggested environment
+- Download the [.NET SDK](https://dotnet.microsoft.com/es-es/download/dotnet/thank-you/sdk-7.0.403-macos-arm64-installer) for your operating system
+- In VSCode, download the .NET extension. Once installed, change the version to 1.26 (the option to change versions is in a dropdown next to the Uninstall button in the extension window)/
+- In VSCode's console, select the Output tab and pick Omnisharp Log in the dropdown.
```
Comment (ncontinanza) [inline README.md:18]: This is not a requirement, but a way of checking that the previous steps worked. If you don't get logs in that tab and you see that Omnisharp is scanning the project, then the config is OK. 
I would rephrase it or leave it separately as a note, so that it doesn't seem as it is a part of the environment set up, what do you think?

**Review by ncontinanza — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (martinacantaro) [reply README.md:18]: Cool! Makes sense!

**Review by ncontinanza — APPROVED** [review]
LGTM!

**Review by Nico-Sanchez — APPROVED** [review]

**Review by ncontinanza — APPROVED** [review]
